tests/test_export_wkw_as_tiff.py

```diff
@@ -0,0 +1,50 @@
+from wkcuber.export_wkw_as_tiff import export_tiff_stack, wkw_name_and_bbox_to_tiff_name, create_parser
+import os
+from PIL import Image
+from wkcuber.mag import Mag
+import wkw
+import numpy as np
+
+def test_export_tiff_stack():
+    parser = create_parser()
+    args_list = ["--source_path", os.path.join("testdata", "WT1_wkw"),
+                 "--destination_path", os.path.join("testoutput", "WT1_wkw"),
+                 "--layer_name", "color",
+                 "--name", "text_export",
+                 "--axis", "z",
```

Comment:
The axis parameter does not exist anymore, right? Also see line 28.

Reply:
That's why the tests failed :D